docs/reference/package-control/repository.md

```diff
@@ -71,14 +71,20 @@ The following values are supported:
 
 ## `packages`
 
-The only required properties for each package are:
+For each package various properties can be specified,
+most importantly `releases` and `details`.
 
-- `details`
-- `releases`
+- `releases`<br>
+  Describes the [releases][] and their requirements.
+- `details`<br>
+  For packages hosted on GitHub, Bitbucket or GitLab,
+  most properties of the package will be retrieved
+  from the `details` URL.
+  Note that for package submissions to the default repository,
+  `details` must always be provided.
```

Comment:
I have no issue with `<br>`, but just in case you were unaware, you can get Markdown to add one with two trailing spaces. It can get annoying for editing and Git to have meaningful trailing whitespace, though. (And that might be why you used `<br>`s.)

Reply:
I am certainly aware and prefer double spaces, but the contribution guide told me not to:

>  Insert manual line breaks using <br> tags instead of using two trailing spaces.